cmd/find-main.go

```diff
@@ -191,13 +220,63 @@ func mainFind(ctx *cli.Context) error {
 	checkFindSyntax(ctx)
 
 	args := ctx.Args()
-
-	if !ctx.Args().Present() {
-		args = []string{"."}
+	if !args.Present() {
```

Comment:
Checks and codes between line#222 and line#227 are already covered as part of the call to `checkFindSyntax(ctx)`. So, they are unnecessary and can be removed.

Reply:
This is needed for argument adjustment.